### PR TITLE
#159520464 Reject Deliveries

### PR DESCRIPTION
**Pull request by nouwatinjacob**
#### What does this PR do?
Add feature for customer to reject deliveries
#### Description of Task to be completed?
- create a reject delivery button on order history
- display a modal with textarea for reason for the rejection
- reject delivery button will only be available when vendor/admin ship the order
#### How should this be manually tested?
- login as a customer
- make order on any product of your choice and checkout
- choose pay on delivery for the method of payment
- login as a vendor
- ship the order
- the customer should visit the order history page
- click on the reject delivery button available at the top of the order details
- a modal will pop to enter the reason for rejection
- the vendor will see that the order as been rejected with the reason for rejection
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#159520464
#### Screenshots or gifs (if appropriate):
screen shot 2018-09-01 at 12 12 29 am
screen shot 2018-09-01 at 12 12 44 am
screen shot 2018-09-01 at 12 16 46 am

#### Questions:

### CODE REVIEW

**Review by otseobande — APPROVED** [review]

**Review by Stylll — APPROVED** [review]
Looks good. Nice job!.

File: imports/plugins/custom/order-cancelation/client/components/Modal.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        >
+          <div className="modal-text-content">
+            <h1>Cancel Order</h1>
+            <h2>Are you sure ?</h2>
+            <div className="modal-ctrl-btn">
+              <button className="btn btn-danger btn-cancel"
+                onClick={e => this.props.cancelOrder(e, this.props.id)}
+              >Yes, I am</button>
+              <button className="btn btn-success btn-cancel"
+                onClick={this.props.toggleCancelModal}
```
Comment (ozimos) [inline imports/plugins/custom/order-cancelation/client/components/Modal.js:56]: use the html code for apostrophe &#39;

Comment (ozimos) [reply imports/plugins/custom/order-cancelation/client/components/Modal.js:56]: the code is &#39

Comment (nouwatinjacob) [reply imports/plugins/custom/order-cancelation/client/components/Modal.js:56]: Thanks  for the feedback. I will correct that

**Pull request merged**